app/konnect-reference-platform/apiops.md

```diff
@@ -0,0 +1,159 @@
+to the platform repository. When the orchestrator's reconciliation loop runs, it will check for changes to the workflow files and stage them as a PR for 
+platform administrators to review and approve. Once the PR is approved, the workflow files are deployed and begin to operate on API specifications
+in the platform repository. The process of initializing the platform repository is covered in the 
+[{{site.konnect_short_name}} Reference Platform how-to guide](/konnect-reference-platform/how-to/).
+
+### How do API Specifications flow through the Reference Platform?
+
+Currently, API Specifications are copied from service repositories to the platform repository
+by the {{site.konnect_short_name}} Orchestrator. This is by design to allow onboarding new services to 
+the platform easily and without disruption to service application teams. In future versions of the reference platform, 
+we will explore extended federated approaches to API specification management.
+
+During the orchestrators reconciliation loop, it will check for changes to API specifications
+in the configured service repositories and stage changed specifications to the platform repository as a PR. 
+When these PRs are reviewed and approved by platform administrators, the APIOps workflows are triggered.
+
+### What are the workflows in the Konnect Reference Platform?
+
+There are 3 main workflows used to deliver APIs to {{site.base_gateway}}. These workflows are completed in
+sequence, with administrator approval between each step. Completion of final workflow results in a deployed
+API to the API gateway.
+
+* **OpenAPI to Kong**: This workflow takes an OpenAPI specification and converts it to 
+  a decK file that can be used by {{site.base_gateway}}. Additionally, service specific patches are
+  applied to the deck file allowing administrators to set overrides for specific APIs.
+* **Stage decK changes**: This workflow takes the decK file generated by the OpenAPI to Kong workflow
+  and compares it to the current state of the {{site.base_gateway}}. It generates a diff and stages the changes
+  as a PR. 
+* **decK sync**: This workflow is triggered by the merging of deck configuration files to the main branch (by the previous workflow PR approval).
+  The workflow triggers a `deck gateway sync` command on the configuration files and applies the changes to {{site.base_gateway}}.
+
+Additionally, the reference platform supports API conformance checks to ensure that specifications are 
+following governance controls and best practices. This is done using the `deck file lint` command 
+which is executed in a GitHub Action workflow. These conformance workflows are run on PRs that contain
+changes to API specifications and deck configuration files. 
+
+You can see examples of these workflows in action in the [Kong Airlines example project](https://github.com/KongAirlines/platform/actions).
```

Comment:
Is it Kong Airlines, KongAirlines, or KongAir? I've seen all three in this PR so far.